src/reports/library/sonatareport.cpp

```diff
@@ -0,0 +1,96 @@
+#include <iostream>
+
+#include "../utils/logger.h"
+#include "element_report.h"
+#include "implementation_interface.hpp"
+#include "soma_report.h"
+#include "sonatareport.h"
+
+namespace bbp {
+namespace sonata {
+
+double SonataReport::atomic_step_ = 1e-8;
```

Comment:
where does 1e-8 come from?

Reply:
Default value I took from the old reportinglib

Reply:
If it's changed, why not make it part of the constructor, and not have it as a static?